src/main/java/lotto/model/LottoRank.java

```diff
@@ -0,0 +1,39 @@
+package lotto.model;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+
+  FIRST(6, 2000000000),
+  SECOND(5, 1500000),
+  THIRD(4, 50000),
+  FOURTH(3, 5000),
+  NON_MATCH(0, 0);
+
+  private int numberOfMatch;
+  private int reward;
```

Comment:
상수 역할을 하는 enum 은 보통 final 키워드를 이용해서 필드의 의미를 더하는데요.
한 번 사용해보시면 좋을 것 같습니다. 😄 
```suggestion
  private final int numberOfMatch;
  private final int reward;
```